apps/mobile/src/screens/signin/signin.screen.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, View } from 'native-base';
+
+import AppStatusBar from '../../shared/components/status-bar/status-bar.component';
+import { useAuth } from '../../shared/hooks/use-auth-provider';
+
+const SignInScreen = () => {
+  const auth = useAuth();
+
+  return (
+    <View flex="1" alignItems="center" justifyContent="center">
+      <AppStatusBar />
+      <Button
+        variant="outline"
+        onPress={() => {
+          auth.signIn({
+            accessToken: '1234',
+            refreshToken: '4321',
```

Comment:
Can we have variables for these values at the top? just to give more context with a meaningful name, it's not clear to me why these values are used here.

Reply:
This is just mock data to simulate the login for now. Later, we will replace it with a call to the backend endpoint.